las/client.py

```diff
@@ -861,9 +874,31 @@ def update_transition_execution(
             'status': status,
             'output': output,
             'error': error,
+            'startTime': f'{start_time}' if start_time else start_time,
         }
         return self._make_request(requests.patch, url, body=dictstrip(body))
 
+    def send_heartbeat(self, transition_id: str, execution_id: str) -> Dict:
+        """Send heartbeat for a manual execution,
```

Comment:
"Send heartbeat for a manual execution to signal that we are still working on it. Must be done at minimum once every 60 seconds or the transition execution will time out"